packages/shared/src/hooks/referral/useJoinReferral.spec.tsx

```diff
@@ -20,7 +20,7 @@ describe('useJoinReferral hook', () => {
           getRedirectUri={jest.fn()}
           updateUser={jest.fn()}
           tokenRefreshed={false}
-          firstLoad={false}
+          isFetched
```

Comment:
`isFetched` can be true when boot fails, so probably should be isFetched and not error

Reply:
Should be updated to use `isError` along with it now.